pkg/types/validation/installconfig_test.go

```diff
@@ -1540,6 +1540,26 @@ func TestValidateInstallConfig(t *testing.T) {
 			}(),
 			expectedError: `capabilities.baselineCapabilitySet: Unsupported value: "vNotValid": supported values: .*`,
 		},
+		{
+			name: "invalid capability marketplace specified without OperatorLifecycleManager",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &types.Capabilities{BaselineCapabilitySet: "None",
+					AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{"marketplace"}}
+				return c
+			}(),
+			expectedError: `additionalEnabledCapabilities: Invalid value: \[\]v1.ClusterVersionCapability{"marketplace"}: the marketplace capability requires the OperatorLifecycleManager capability`,
+		},
+		{
+			name: "invalid capability baremetal specified without MachineAPI",
```

Comment:
Nit: This test seems unrelated to the current PR.

Reply:
Apologies, I noticed we were missing one for that case, so I figured it wouldn't hurt to add one since they're a similar code path